### PR TITLE
Updated channel and ifo submission format

### PR DESCRIPTION
**Pull request by wbenoit26**
Changes the way that ifo and channel information is formatted in our gracedb submissions to align with what Alex and Roberto are requesting. I need the gracedb PR to be merged before I can test this.

### CODE REVIEW

**Comment by EthanMarx** [comment]
Looks good - have tested submitting with this to GraceDB?

**Comment by wbenoit26** [comment]
@EthanMarx I tested this out and the format is showing up as expected: [https://gracedb-test.ligo.org/events/G1188124/view/](https://gracedb-test.ligo.org/events/G1188124/view/)

File: projects/online/config.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ amplfi_architecture:
             groups: 8
 amplfi_weights: ${oc.env:AMPLFI_WEIGHTS}
 ifos: ["H1", "L1"]
-channel: "GDS-CALIB_STRAIN"
+channel: ["H1:GDS-CALIB_STRAIN", "L1:GDS-CALIB_STRAIN"]
```
Comment (EthanMarx) [inline projects/online/config.yaml:24]: I think this should be `channels`

**Review by EthanMarx — CHANGES_REQUESTED** [review]
One nit pick on the config. looks great

File: projects/online/config.yaml (hunk 1 above)
Comment (wbenoit26) [reply projects/online/config.yaml]: Good catch, fixed

**Pull request merged**